src/priorityQueue.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+class Node {
+  constructor(follows, data, priority, proceeds) {
+    this.after = follows
```

Comment:
Trying to use unconventional names to make sure I understand the code and how things are "shifting" around, but making parameters the same as properties names is agreed.